action.yml

```diff
@@ -48,6 +48,18 @@ inputs:
     description: Whether to force the installation of jq
     required: false
     default: "true"
+  sort-by:
+    required: false
+    description: 'Sort by query'
+    default: '.stack_slug'
+  group-by:
+    required: false
+    description: 'Group by query'
+    default: '.stack_slug | split("-") | [.[0], .[2]] | join("-")'
```

Comment:
I think this will fail for customers who aren't using tenant (some of the older customers) and for customers who have changed the `name_pattern` in `atmos.yaml`

Reply:
They can override it in their workflows

Reply:
That's why we have this input configurable